src/ParcelRegistry.Api.Oslo/Properties/launchSettings.json

```diff
@@ -0,0 +1,26 @@
+{
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:13215/",
+      "sslPort": 0
+    }
+  },
+  "profiles": {
+    "IIS Express": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "ParcelRegistry.Api": {
```

Comment:
`ParcelRegistry.Api.Oslo` or `ParcelRegistry.Oslo` will be less confusing for the legacy profile.